packages/tofu/package.json

```diff
@@ -5,10 +5,14 @@
   "license": "MIT",
   "dependencies": {
     "@babel/parser": "^7.23.0",
-    "@babel/traverse": "^7.23.2"
+    "@babel/traverse": "^7.23.2",
+    "@babel/types": "^7.23.3",
+    "@types/babel__traverse": "^7.20.4",
```

Comment:
so we need both? won't `@babel/types` define that already?

Reply:
`@types/babel__traverse` depends upon `@babel/types`, but they aren't the same.